src/transformers/models/oneformer/modeling_oneformer.py

```diff
@@ -193,13 +193,13 @@ def pair_wise_sigmoid_cross_entropy_loss(inputs: torch.Tensor, labels: torch.Ten
     height_and_width = inputs.shape[1]
 
     criterion = nn.BCEWithLogitsLoss(reduction="none")
-    cross_entropy_loss_pos = criterion(inputs, torch.ones_like(inputs))
-    cross_entropy_loss_neg = criterion(inputs, torch.zeros_like(inputs))
+    # Prevent overflow when using torch 16bit autocast
```

Comment:
It might not be autocast - in some cases we explicitly set this   

```suggestion
    # Prevent overflow when using torch 16bit
```